build_cmake_pybullet_double.sh

```diff
@@ -5,7 +5,7 @@ if [ -e CMakeCache.txt ]; then
 fi
 mkdir -p build_cmake
 cd build_cmake
-cmake -DBUILD_PYBULLET=ON -DBUILD_PYBULLET_NUMPY=ON -DUSE_DOUBLE_PRECISION=ON -DBT_USE_EGL=ON -DCMAKE_BUILD_TYPE=Release .. || exit 1
+cmake -DBUILD_PYBULLET=ON -DBUILD_PYBULLET_NUMPY=ON -DUSE_DOUBLE_PRECISION=ON -DBT_USE_EGL=ON -DCMAKE_BUILD_TYPE=Release -DPYTHON_EXECUTABLE:FILEPATH=/opt/homebrew/Caskroom/miniforge/base/bin/python3 -DPYTHON_INCLUDE_DIR:PATH=/opt/homebrew/Caskroom/miniforge/base/include/python3.9 -DPYTHON_LIBRARY:FILEPATH=/opt/homebrew/Caskroom/miniforge/base/lib/libpython3.9.dylib .. || exit 1
```

Comment:
please revert this file or exclude it from your PR.

Reply:
will do